src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -103,6 +115,7 @@ public void deleteMessage(String ID) {
   public List<Message> getAllMessages(){
```

Comment:
Nit: It might be clearer to rename this to getAllPublicMessages? But it's also fine to leave it.

It would also be good to update the javadoc!

Reply:
done